.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Done Criteria Pull Request
+
+## Issue #11 https://github.com/Access-Point-Program/cohort-3-simulation-runner/issues/11
```

Comment:
I would remove this line from the Pull Request template. 

If you are planning to have each Pull Request linked to an issue then you can keep this. However, the way that it is not, it will link all Pull requests to Issue #11 automatically which is not what you want. 

My suggestion would be 
```suggestion
## Issue #<-- replace this is the issue number you are working on -->
```